### PR TITLE
Performance improvement to filterFn

### PR DESCRIPTION
**Pull request by RNCRyan**
Changed out string.match(RegExp) for RegExp.test on the filter function.
It's faster.

### CODE REVIEW

File: paper-typeahead.js [hunk 1]
```diff
@@ -24,7 +24,7 @@
             }
 
             return data.filter(function(v) {
-              return v.match(r);
+			  return (r.test(v) ? v : null);
```
Comment (samccone) [inline paper-typeahead.js]: awesome @RNCRyan!

It looks like you are using tabs here, would you mind switching to spaces? thx :sparkles:

**Comment by RNCRyan** [comment]
Sorry about that.  Is it good to go now, or do you need this squashed?

**Comment by samccone** [comment]
if you would not mind a squash :)

**Comment by RNCRyan** [comment]
Alright, check it out one more time.

**Comment by samccone** [comment]
:clap: thanks a bunch!

**Pull request merged**